docs/usage/faq.mdx

```diff
@@ -11,14 +11,25 @@ description: Common questions about OpenHands and their answers
 
 ### It's doing stuff without asking, is that safe?
 
-**Yes, it's safe!** OpenHands runs all code in a secure, isolated Docker container (called a "sandbox") that is completely separate from your host system. This means:
+**Generally yes, but with important considerations.** OpenHands runs all code in a secure, isolated Docker container (called a "sandbox") that is separate from your host system. However, the safety depends on your configuration:
 
-- **No access to your host system**: The agent cannot access files, programs, or data outside the container
-- **Isolated environment**: Any code execution happens only within the container
-- **Resource control**: The container has limited resources and cannot affect your host machine's performance
-- **Easy cleanup**: You can stop or delete the container at any time without affecting your system
+**What's protected:**
+- Your host system files and programs (unless you mount them)
+- Host system resources and performance
+- Other containers and processes
 
-The sandbox architecture is specifically designed to allow AI agents to work autonomously while keeping your system secure. Learn more about how this works in our [Runtime Architecture documentation](/usage/architecture/runtime).
+**Potential risks to consider:**
+- The agent can access the internet from within the container
+- If you provide credentials (API keys, tokens), the agent can use them
+- Mounted files and directories can be modified or deleted
+- Network requests can be made to external services
+
+**Security features:**
+- Built-in security analyzer to detect potentially risky actions
+- Configurable confirmation mode for sensitive operations
+- Hardened Docker configurations for production use
```

Comment:
@OpenHands we should note that this security feature comes from community contribution and is not by default enabled